gcp/rakefiles/vars.rb

```diff
@@ -67,6 +68,9 @@ def self.set_vars(env, project_type)
       ENV['TF_VAR_dataloader_checksum'] = versions['gpii-dataloader'].split('@')[1]
     end
   end
+
+  # Hack to force Terraform to reapply some resources on every run
+  ENV["TF_VAR_nonce"] = SecureRandom.hex
```

Comment:
Please add a test for this :)

Also, be careful when merging with #44 as this belongs in `set_vars`, not `set_versions`.

Reply:
Done